src/bci_build/package/python/README.md.j2

```diff
@@ -0,0 +1,56 @@
+# {{ image.pretty_name }} container image
+
+{% include 'badges.j2' %}
+
+## Description
+
+[Python](https://www.python.org/) is an interpreted, interactive, object-oriented, open-source programming language. It incorporates modules, exceptions, dynamic typing, high-level dynamic data types, and classes. It provides interfaces to many system calls, libraries, and various window systems, and it is extensible in C or C++. It is also usable as an extension language for applications that require programmable interfaces.
+
+## Usage
+
+To deploy an application, install dependencies, copy the sources, and configure the application's main script:
+
+```Dockerfile
+FROM {{ image.pretty_reference }}
+
+WORKDIR /app
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD [ "python3", "./main-script.py" ]
+```
+
+Build and run the container image:
+
+```ShellSession
+$ podman build -t my-python-app .
+$ podman run -it --rm my-python-app
+```
+
+To run a single script inside a container, use the following command:
+
+```ShellSession
+$ podman run --rm -v "$PWD":/app:Z -w /app {{ image.pretty_reference }} python3 script.py
+```
+
+## Additional tools
+
+The following tools are included in the image:
```

Comment:
maybe this shouldn't say "following tools" and then a list of package names. I think we could simply have two paragraphs that explain what "pip" is (if pip is available, which is always the case) and what pipx is (if pipx is available in the image).

Reply:
Instead of package name, I was considering having the tool name (e.g not python3-pipx, just pipx) and adding a small description.

- pipx — Install and Run Python Applications in Isolated Environments

@dirkmueller @dcermak WDYT? this at least fix the problem with including the package version

Reply:
one thing we discussed is the usefulness of this section, because of this section this PR is stuck. Perhaps removing this completely and replacing it with a # pipx and # else section is better.

Reply:
I think we can just mention that pip (& pipx) is/are installed, or just dropping it entirely

Reply:
I kept this section, but I changed it to include tool names, it seems more useful.